responcive page

ali1373107 opened the pull request.

ali1373107 committed:
first commit

ali1373107 committed:
ddd

Hamisakim commented on index.html:
```diff
@@ -2,20 +2,96 @@
-
-    <!-- Please leave this in to credit the creators of this design -->
-    <p>This website design was created by Wix.com, and is used here for strictly educational purposes.</p>
+    <header>
```
keep nav bar seperate from header

Hamisakim commented on index.html:
```diff
@@ -2,20 +2,96 @@
+            </section>
+            
+        </main>
+        <footer>
```
keep footer outside content

ali1373107 committed:
sundays commit

ali1373107 committed:
final commit

I did add position to which element was needed. I havent add hover to buttons yet

ali1373107 commented:
Hi Sami this is the latest commit that I have made

ali1373107 commented:
I havent added hover to elements yet

Hamisakim commented:
good work @ali1373107.  Going forwards lets make sure to focus more little bits of styline. Like the font, this will make the website much nicer to look at.
